docs/platform/what-is-tizen/overview.md

```diff
@@ -46,6 +46,13 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
```

Comment:
"MediaVision" should be written as a third heading to make it distinguishable from other third headings under the "AI and Machine Learning" section.
Ref: In other PRs (https://github.com/Samsung/tizen-docs/pull/1616 & https://github.com/Samsung/tizen-docs/pull/1611), different authors are working on different sections under the "AI and Machine Learning" section.
```suggestion
### MediaVision
```